scripts/git_hooks/README.md

```diff
@@ -0,0 +1,2 @@
+# Configure git to use the new hook - it will stay with the repository
+git config core.hooksPath "./scripts/git_hooks"
```

Comment:
Might be good to tell what's the directory contains and what's running - ppl can check of course, but let's be precise :).

Reply:
done (as a makefile rule)